allen-cell-animated/cell-feature-explorer
# Feature/channel rename groupings

Problem
=======
the 3d viewer has been updated to receive channel name remappings and groupings as props.  We also want to pick up other good updates from the latest viewer.

Solution
========
Update the viewer version and specify the channel name remappings and groupings

## Type of change

* Bug fix (non-breaking change which fixes an issue)
* New feature (non-breaking change which adds functionality)

Steps to Verify:
----------------
Load variance dataset and confirm that channel names in 3d viewer are the same as in the deployed version of CFE.
